src/run-on-blame.ts

```diff
@@ -17,6 +17,9 @@ export async function runOnBlame(files: string[]): Promise<void> {
       core.getInput('phpcs_path', { required: true })
     );
 
+    const dontFailOnError =
+      core.getInput('fail_on_errors') == 'false' ||
+      core.getInput('fail_on_errors') === 'off';
```

Comment:
Not really great to use double negatives in variable names, but given that `dontFailOnWarning` already exists, meh.
Although given that this is later only used as `!dontFailOnError` I think it's actually better to simply name it `failOnError`.

Reply:
I agree but the original author already had this, I only intended to add a "copy" which works the same as the warning option.